Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -973,7 +973,12 @@ private void ReleaseRigidBody(Vector3 velocity, Vector3 angularVelocity)
             {
                 rigidBody.useGravity = wasGravity;
                 rigidBody.isKinematic = wasKinematic;
+            }
```

Comment:
I think the blow code needs to stay in this `if` block, so we don't try to manipulate a null `rigidBody`